hud-pages/m0re-techno.md

```diff
@@ -1,13 +1,20 @@
 # Description
 
-my current tf2 hud as of 18/01/23 based on dqz's version of m0rehud 
+<a href=https://github.com/TechnoSL/m0re_techno/blob/master/fonts.md>this hud has multiple fonts, click here to see previews!</a>
```

Comment:
Use markdown for links: `[text](https://example.com)`

Reply:
done, thanks for letting me know